### PR TITLE
chore: repairs in categories taxonomy - week 39

### PR DESCRIPTION
**Pull request by aleene**
### What
repairs in categories taxonomy

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -48613,10 +48613,10 @@ fr:Yaourts brassés, Yaourt brassé
 nl:Roeryoghurts
 #wikidata:en:
 
-<ru:Йогурт
+<en:Yogurts
 ru:Биойогурт, Биойогурты
 
-<ru:Йогурт
+<en:Yogurts
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:48619]: I think this category is a synonyme of "Fruit yogurts"

**Review by moon-rabbitOFF — APPROVED** [review]

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -102901,7 +102940,7 @@ fr:Palmiers salés
 nl:Gezouten vlinders, Gezouten palmiers
 
 <en:Crackers
-fr:Crêpes dentelle salées, Crêpes dentelles salées
+fr:Crêpes dentelles salées, Crêpes dentelles salées
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:102943]:
```suggestion
fr:Crêpes dentelle salées, Crêpes dentelles salées
```
Dentelle stays singular when crêpe is plural. French language makes no sense sometimes

**Review by moon-rabbitOFF — CHANGES_REQUESTED** [review]

File: taxonomies/categories.txt (hunk 2 above)
Comment (aleene) [reply taxonomies/categories.txt]: ok imolemented

File: taxonomies/categories.txt (hunk 1 above)
Comment (aleene) [reply taxonomies/categories.txt]: OK

**Comment by aleene** [comment]
@stephane I am a bit stuck with resolving this. Should some changes be reverted?

**Comment by stephane** [comment]
Wrong person ;)

**Comment by aleene** [comment]
@stephanegigandet ?

**Comment by alexgarel** [comment]
@aleene do you still need @stephanegigandet help here ? You might ping him on slack !

**Comment by aleene** [comment]
Will have a look next week when im home again

**Comment by stephanegigandet** [comment]
Looking into it

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**